docker/init-db.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
copied from https://github.com/MarquezProject/marquez/blob/master/docker/init-db.sh - we could perhaps improve this flow